spec/c04matrices/calculate/find_the_sum_of_diagonal_right_matrixSpec.js

```diff
@@ -0,0 +1,14 @@
+const sumRightDiagonals = require('../../../src/c04matrices/calculate/find_the_sum_of_diagonal_right_matrix')
+
+describe('Sum of the right diagonals of a matrix', function () {
+
+    it('Should find the right diagonals of matrix and calculate the sum', function () {
+        let matrix = [[1,2],
```

Comment:
This test is not a good test. Sum of Left diagonal is same as sum of right diagonal.

Reply:
i maked the test with exemple from problem   /*Write a program to find the sum of the right diagonals of a matrix.
Test Data :
Input the size of the square matrix : 2
Input elements in the first matrix :
element - [0],[0] : 1
element - [0],[1] : 2
element - [1],[0] : 3
element - [1],[1] : 4
Expected Output :
The matrix is :
1 2
3 4
Addition of the right Diagonal elements is :5 */